app/managers/websocket_manager.ts

```diff
@@ -92,23 +99,21 @@ class WebsocketManager {
     public closeAll = () => {
         for (const url of Object.keys(this.clients)) {
             const client = this.clients[url];
-            if (client.isConnected()) {
-                client.close(true);
-                this.getConnectedSubject(url).next('not_connected');
-            }
+            client.close(true);
+            this.getConnectedSubject(url).next('not_connected');
         }
     };
 
     public openAll = async () => {
+        let queued = 0;
         for await (const clientUrl of Object.keys(this.clients)) {
             const activeServerUrl = await DatabaseManager.getActiveServerUrl();
             if (clientUrl === activeServerUrl) {
                 this.initializeClient(clientUrl);
             } else {
+                queued += 1;
```

Comment:
We used to connect all other servers at the same time, because the debounce was not really doing what was planned. Now it does what it should.

Reply:
Not sure if something like a peiority queue is needed but we could build a poc. Perhaps we can get sole benefits from it

Reply:
We could consider this, but I feel it is out of the scope of this PR.